tex/set-theory/forcing.tex

```diff
@@ -499,7 +545,9 @@ \section{(Optional) Defining the relation}
 		\left( q \le r \land q \Vdash(\tau=\tau_1) \right) \right\}
 \]
 is dense below in $p$.
-In other words, if the set is dense, then the generic must hit $q$, so it must hit $r$, meaning that $\left<\tau_r\right> \in \tau_2$ will get interpreted such that $\tau^G \in \tau_2^G$, and moreover the $q \in G$ will force $\tau_1 = \tau$.
+In other words, if the set is dense, then the generic must hit $q$, so it must hit $r$
+(recall that a filter is upwards-closed),
```

Comment:
might be too explicit, but I'm probably not the only one to forget it. (?)

Reply:
as a rule of thumb i think err on the side of more explanation, it's much easier to skip over overly verbose sentences than fill in any gaps.